index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Login page</title>
+    <link rel="stylesheet" href="global.css">
+</head>
+<body id="index">
+    <div class="container">
+
+        <h1 id="myH1Login">Snake Game</h1>
+
+        <p>Use arrow keys for playing</p>
+        <p>⬆️</p>
+        <p>⬅️⬇️➡️ </p>
+
+        <a href="home.html"><button type="button">Start</button></a>  
+    </div>
+
+    <script src="index.js"></script>
```

Comment:
Scripts shall always be under the body as a good practice to follow and use defer too.